frontend/src/components/ViewTrips/trip.js

```diff
@@ -6,24 +6,25 @@ import { timestampToISOString } from '../Utils/time.js';
 import { getUserEmailArrFromUserUidArr } from '../Utils/temp-auth-utils.js';
 import DeleteTripButton from './delete-trip-button.js';
 import ViewActivitiesButton from './view-activities-button.js';
+import * as DB from '../../constants/database.js';
 
 /**
  * Returns the string date range of the trip associated with the Trip document
- * data `tripObj`.
+ * start and end date timestamps.
  *
- * Notes:
- *  - tripObj will always contain valid start_date and end_date fields.
- *  - When the Firestore Timestamps contained in `tripObj` converted to js
+ * Note: When the Firestore Timestamps contained in `tripObj` converted to js
  *    dates, the months are 0 indexed rather than 1 indexed so they must be
  *    incremented by 1 in order for the month to be correct.
  *
- * @param {!firebase.firestore.DocumentData} tripData Object containing the
- *     fields and values for a Trip document.
- * @return {string} Date range of the trip.
+ * @param {!firebase.firestore.Timestamp} startDateTimestamp Firestore timestamp
+ *     Object corresponding to the trip start date.
+ * @param {!firebase.firestore.Timestamp} endDateTimestamp Firestore timestamp
+ *     Object corresponding to the trip end date.
+ * @return {string} Date range of the trip in the form 'MM/DD/YYY - MM/DD/YYY'.
  */
-export function getDateRange(tripData) {
-  const startDate = tripData.start_date.toDate();
-  const endDate = tripData.end_date.toDate();
+export function getDateRange(startDateTimestamp, endDateTimestamp) {
```

Comment:
this function is another friend to time.js, if you'd like to share :)

Reply:
Yeah I will plop it in there even though it is pretty simple at this point.